go.mod

```diff
@@ -11,6 +11,8 @@ require (
 	github.com/openshift/library-go v0.0.0-20200910214143-887092e305c1
```

Comment:
This file should end up in the vendoring commit. :frowning_face:

Reply:
Done.